EIPS/eip-5792.md

```diff
@@ -12,86 +12,42 @@ created: 2022-10-17
-
-### `wallet_getCapabilities`
-
-RPC for an application to receive information about the capabilities that a provided wallet supports (e.g. batch transactions, paymaster communication).
-
-This method SHOULD return an error if the user has not already approved a connection to the app with the provided address.
-
-The key of each item per chain is the name of a capability and the value can be of any shape. Capabilities are returned per chain because wallets may support different capabilities across chains.
-
-We expect the community to align on the definition of capabilities in separate ERCs.
-
-#### `wallet_getCapabilities` RPC Specification
-
-```typescript
-type GetCapabilitiesParams = [`0x${string}`]; // Wallet address
-
-type GetCapabilitiesResult = Record<`0x${string}`, <Record<string, any>>; // Hex chain id
-```
-
-##### `wallet_getCapabilities` Example Parameters
-
-```json
-["0xd46e8dd67c5d32be8058bb8eb970870f07244567"]
-```
-
-##### `wallet_getCapabilities` Example Return Value
-
-The capabilities below are for illustrative purposes.
-
-```json
-{
-  "0x2105": {
-    "paymasterService": {
-      "supported": true
-    },
-    "sessionKeys": {
-      "supported": true
-    }
-  },
-  "0x14A34": {
-    "paymasterService": {
-      "supported": true
-    }
-  }
-}
-```
+We also define one capability expression for use with the fourth method, which further enables a wallet to promise atomicity of the execution of calls passed and managed by the first three methods.
 
 ### `wallet_sendCalls`
 
-Requests that a wallet submits a batch of calls. From and chain ID are top level because all calls should be sent from the same sender on the same chain. The items in the calls field are only those that are shared by all transaction types. Any other fields that a wallet may need to submit a transaction should be handled by the wallet. 
+Requests that a wallet submits a batch of calls. `from` and `chainId` are top-level properties rather than per-call properties because all calls should be sent from the same sender and on the same chain, identified by [EIP-155](./eip-155.md) integers expressed in hexidecimal notation. The items in the `calls` field are only those that are shared by all transaction types. Any other fields that a wallet may need to submit a transaction should be handled by the wallet.
 
-The capabilities field is how an app can communicate with a wallet about capabilities a wallet supports. For example, this is where an app can specify a paymaster service URL from which an [ERC-4337](./eip-4337.md) wallet can request a `paymasterAndData` input for a user operation.
+The capabilities field is how an app can communicate with a wallet about its own capabilities, which a wallet may or may not support. For example, this is where an app can specify a paymaster service URL from which an [ERC-4337](./eip-4337.md) wallet can request a `paymasterAndData` input for a user operation if the wallet supports that kind of paymaster service.
```

Comment:
`The capabilities field is how an app can communicate with a wallet about its own capabilities, which a wallet may or may not support.` I think the original wording was more accurate -- the `capabilities` field is indeed for communication about the _wallet's_ capabilities. Let's revert this sentence?